### PR TITLE
Feature/scenario scale kubernetes cluster

### LINKED ISSUE #133: Remove ActionList
It should be possible to execute actions right away instead of storing the in an ActionList and executing later

### PR DESCRIPTION
**Pull request by proman3419**
Related to https://github.com/k8loud/demo/issues/10
Closes #133 

[Added GetServerNamesAction](https://github.com/k8loud/themis-executor/commit/01d2b4141e46c5623ec4842cf0ace8a03116e7d5)
[Added DetachNodeFromKubernetesClusterAction](https://github.com/k8loud/themis-executor/commit/80f214ba72156092d594d3aa9bdc1a8263612933)
[Modified DeleteInstanceAction such that a namePattern can be passed](https://github.com/k8loud/themis-executor/commit/60a8471ec8472ca962807184bd792b0c549a8cc9)
[Async DroolsService](https://github.com/k8loud/themis-executor/commit/c06aefb9dccff94fabbed99fa41c96c4c880b22c)
[Stateful CronChecker](https://github.com/k8loud/themis-executor/commit/2fb031dc3f4cb08e9fe1e42e34dce8797f6a9158)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ DOCKERHUB_USERNAME=k8loud
 IMAGE_NAME=themis-executor
 # w.x.y.z, one digit value each
 # when tinkering add -<description> suffix
-VER=0.0.3.4
+VER=0.0.3.4-scenario-scale-kubernetes-cluster
```
Comment (pgorgolew) [inline Makefile:7]: version

File: manifests/05-deployment.yaml [hunk 2]
```diff
@@ -19,7 +19,7 @@ spec:
       serviceAccountName: themis-executor
       containers:
         - name: executor
-          image: k8loud/themis-executor:0.0.3.4
+          image: k8loud/themis-executor:0.0.3.4-scenario-scale-kubernetes-cluster
```
Comment (pgorgolew) [inline manifests/05-deployment.yaml:22]: version

File: src/main/java/org/k8loud/executor/drools/DroolsService.java [hunk 3]
```diff
@@ -84,35 +84,24 @@ private StatelessKieSession createSession() {
         return kieSession;
     }
 
+    @Async
```
Comment (pgorgolew) [inline src/main/java/org/k8loud/executor/drools/DroolsService.java:87]: is async needed?

**Review by pgorgolew — CHANGES_REQUESTED** [review]

File: src/main/java/org/k8loud/executor/drools/DroolsService.java (hunk 3 above)
Comment (proman3419) [reply src/main/java/org/k8loud/executor/drools/DroolsService.java:87]: https://www.baeldung.com/spring-scheduled-tasks#schedule-a-task-at-a-fixed-rate
```
Note that scheduled tasks don’t run in parallel by default. 
So even if we used fixedRate, the next task won’t be invoked until the previous one is done.
If we want to support parallel behavior in scheduled tasks, we need to add the @Async annotation
```

**Review by pgorgolew — APPROVED** [review]

**Pull request merged**